Content.Server/Administration/Systems/SuperBonkSystem.cs

```diff
@@ -31,7 +30,7 @@ public void StartSuperBonk(EntityUid target, float delay = 0.1f, bool stopWhenDe
                 return;
         }
 
-
+        // Permanently give the clumsy component.
```

Comment:
> Why did you change this to be permanent?

I think I changed it because it was broken before. I did ask the admins if it was OK and they said it was fine. Why I didn't remove all the other stuff is a mystery to me 😆